persistence/src/main/java/edu/unc/lib/dl/persist/services/destroy/DestroyObjectsJob.java

```diff
@@ -125,8 +127,23 @@ public void run() {
                 }
 
                 if (!repoObj.getResource().hasProperty(RDF.type, Cdr.Tombstone)) {
+                    RepositoryObject parentObj = repoObj.getParent();
+
                     // purge tree with repoObj as root from repository
+                    // Add the root of the tree to delete
+                    deletedObjCount += 1;
+                    deletedObjIds.add(repoObj.getPid().getUUID());
+
                     destroyTree(repoObj);
+
+                    // Add premis event to parent
+                    parentObj.getPremisLog().buildEvent(Premis.Deletion)
+                            .addAuthorizingAgent(agent.getUsername())
+                            .addOutcome(true)
+                            .addEventDetail("{0} object(s) were destroyed", deletedObjCount)
+                            .addEventDetail("Objects destroyed:\n {0}",
```

Comment:
Remove the space before `{0}` so that all of the entries have the same indentation.
Also, for the sake of consistency you should use `System.getProperty("line.separator")` here too. You're welcome to it in a variable if you want to shorten things, etiher way is fine.